mappings/net/minecraft/client/render/block/entity/ChestBlockEntityRenderer.mapping

```diff
@@ -7,3 +13,12 @@ CLASS net/minecraft/class_826 net/minecraft/client/render/block/entity/ChestBloc
 	FIELD field_4364 NORMAL_TEX Lnet/minecraft/class_2960;
 	FIELD field_4365 isChristmas Z
 	FIELD field_4366 ENDER_TEX Lnet/minecraft/class_2960;
+	METHOD method_22749 bufferModel (Lnet/minecraft/class_287;Lnet/minecraft/class_630;Lnet/minecraft/class_630;Lnet/minecraft/class_630;FIILnet/minecraft/class_1058;)V
```

Comment:
Does 'buffering' make sense as the action happening here? I don't know what the method does so I'm not sure, just seems odd

Reply:
The chest block entity now "renders" to a buffer (no direct gl calls) which is later rendered by minecraft itself.

Reply:
Yeah, but 'buffering' isn't the action -- I think it'd make sense for the action part to be like 'build' or 'render'

Reply:
Perhaps `renderToBuffer`?

Reply:
That's good